Services/Authorization/PsychologicalSupportPlatform.Authorization.API/Controllers/AccountsController.cs

```diff
@@ -2,38 +2,39 @@
 using Microsoft.AspNetCore.Mvc;
 using PsychologicalSupportPlatform.Authorization.Application.Interfaces;
 using PsychologicalSupportPlatform.Authorization.Domain.DTOs;
-using PsychologicalSupportPlatform.Authorization.Domain.Entities;
 using PsychologicalSupportPlatform.Common;
 
 namespace PsychologicalSupportPlatform.Authorization.API.Controllers
 {
     [Route("api/[controller]")]
     [ApiController]
-    public class LoginController : ControllerBase
+    public class AccountsController : ControllerBase
     {
-        private readonly ILoginService loginService;
+        private readonly IAccountsService _accountsService;
 
-        public LoginController(ILoginService loginService)
+        public AccountsController(IAccountsService accountsService)
         {
-            this.loginService = loginService;
+            _accountsService = accountsService;
         }
 
-        [HttpPost]
-        [Route("login")]
+        [HttpPost("login")]
         public async Task<IActionResult> LoginAsync(LoginData data)
         {
-            var response = await loginService.GetTokenAsync(data);
+            var response = await _accountsService.GetTokenAsync(data);
 
-            if (response.Data is null) return Unauthorized(response.Message);
+            if (response.Data is null)
```

Comment:
read about result pattern. you can define model with the property that will hold the actual data and status property. status property will be enum with different status codes. you will return this model from both service and controller (maybe in controller you will need to use ToActionResult() method to return such model). and property with status code will eliminate the need of all checks and returns of Ok, Unauthorized and etc. in contoller.

Reply:
Changed ResponseInfo and DataResponseInfo classes